www/app/login/login.controller.js

```diff
@@ -20,14 +20,15 @@
     activate();
 
     function activate() {
+      LoginFirebaseService.verifyAccess();
       $ionicNavBarDelegate.showBackButton(false);
     }
 
     function _authSuccess() {
       $state.go('scannerMenu');
     }
 
-    function authError() {
+    function _authError() {
       $ionicPopup.alert({
         title: ' Error de Autenticación',
         template: ' Autenticación Invalida'
```

Comment:
Nice way to define private functions, but we have to keep the same codestyle for every file. So we either change the name in the other private functions in the project or we keep them the same.